composeApp/src/androidMain/kotlin/com/wisemen/chargehub/ui/screens/LandingScreen.kt

```diff
@@ -0,0 +1,129 @@
+package com.wisemen.chargehub.ui.screens
+
+import androidx.compose.foundation.Image
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.width
+import androidx.compose.material3.ButtonDefaults
+import androidx.compose.material3.Icon
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.LaunchedEffect
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.layout.ContentScale
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import androidx.navigation.NavController
+import com.ramcosta.composedestinations.annotation.Destination
+import com.wisemen.chargehub.R
+import com.wisemen.chargehub.nav.ChargeHubNavGraph
+import com.wisemen.chargehub.ui.components.Buttons
+import com.wisemen.chargehub.ui.components.primaryButtonColors
+import com.wisemen.chargehub.ui.components.secondaryButtonColors
+import com.wisemen.chargehub.ui.theme.AppTheme
+import com.wisemen.chargehub.ui.theme.Colors
+import com.wisemen.chargehub.ui.theme.TextStyles
+import org.koin.java.KoinJavaComponent.inject
+import screens.landing.LandingScreenUiAction
+import screens.landing.LandingScreenUiEvent
+import screens.landing.LandingScreenViewModel
+
+@Preview
+@Composable
+fun LandingScreenPreview() {
+    AppTheme {
+        LandingLayout {}
+    }
+}
+
+@Composable
+@Destination
+@ChargeHubNavGraph(start = true)
+fun LandingScreen(
+    navController: NavController
+) {
+    val viewModel: LandingScreenViewModel by inject(LandingScreenViewModel::class.java)
+
+    LaunchedEffect(viewModel) {
+        viewModel.eventFlow.collect { event ->
+            when (event) {
+                is LandingScreenUiEvent.ClickedLoginButtonEvent -> {}
+                is LandingScreenUiEvent.ClickedRegisterButtonEvent -> {}
+            }
+        }
+    }
+    LandingLayout(viewModel::onAction)
+}
+
+@Composable
+fun LandingLayout(onAction: (LandingScreenUiAction) -> Unit) {
+    Box(modifier = Modifier.fillMaxSize()) {
+        Image(
+            painter = painterResource(R.drawable.landingbackground),
+            contentDescription = "background img",
+            contentScale = ContentScale.FillBounds,
+            modifier = Modifier.matchParentSize()
+        )
+        Column(
+            modifier = Modifier.padding(top = 46.dp, start = 16.dp, end = 16.dp)
```

Comment:
Just a small remark, i would place the padding top onto your header so that you column takes up the whole screen and you could put a horizontal padding like: Padding.medium or something like that